File: app/src/main/java/com/fjsdkqwj/pfdioewjnsd/ui/AppInfoActivity.java
package com.fjsdkqwj.pfdioewjnsd.ui;

import android.view.View;
import android.view.WindowManager;
import android.widget.TextView;

import com.fjsdkqwj.pfdioewjnsd.R;
import com.fjsdkqwj.pfdioewjnsd.base.BaseActivity;
import com.fjsdkqwj.pfdioewjnsd.util.SharePreferencesUtil;
import com.fjsdkqwj.pfdioewjnsd.util.StaticUtil;
import com.fjsdkqwj.pfdioewjnsd.util.StatusBarUtil;

public class AppInfoActivity extends BaseActivity {

    private TextView appVersionInfoTv, titleTv;
    private View backImg;

    @Override
    public int getLayoutId() {
        return R.layout.activity_app_info;
    }

    @Override
    public void initListener() {
        backImg.setOnClickListener(v ->{
            finish();
        });
    }

    @Override
    public void initData() {
        if (SharePreferencesUtil.getBool("NO_RECORD")) {
            getWindow().addFlags(WindowManager.LayoutParams.FLAG_SECURE);
        }
        StatusBarUtil.setTransparent(this, false);
        appVersionInfoTv = findViewById(R.id.app_version_info_tv);
        titleTv = findViewById(R.id.title_tv);
        backImg = findViewById(R.id.back_image);
        titleTv.setText("关于我们");
        appVersionInfoTv.setText("当前版本号：v" + StaticUtil.getAppVersion(this));
    }
}

File: app/src/main/java/com/asvsdfer/new_master_code_kotlin/ui/activity/AppInfoActivity.kt
package com.asvsdfer.new_master_code_kotlin.ui.activity

import android.content.Context
import android.content.pm.PackageManager
import android.os.Bundle
import android.text.TextUtils
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity
import com.asvsdfer.new_master_code_kotlin.R
import com.asvsdfer.new_master_code_kotlin.util.PreferencesOpenUtil
import com.asvsdfer.new_master_code_kotlin.util.StatusBarUtil
import kotlinx.android.synthetic.main.activity_app_info.*
import kotlinx.android.synthetic.main.top_layout.*

class AppInfoActivity: AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        StatusBarUtil.setTransparent(this, false)
        setContentView(R.layout.activity_app_info)
        if (PreferencesOpenUtil.getBool("NO_RECORD") == true) {
            window.addFlags(WindowManager.LayoutParams.FLAG_SECURE)
        }
        title_tv.text = "关于我们"
        app_version_info_tv.text = "当前版本号：v" + getAppVersion(this)
        back_image.setOnClickListener { finish() }
    }

    fun getAppVersion(context: Context): String? {
        var version = ""
        try {
            val packageManager = context.packageManager
            val packageInfo = packageManager.getPackageInfo(context.packageName, 0)
            version = packageInfo.versionName
            if (TextUtils.isEmpty(version) || version.length <= 0) {
                return ""
            }
        } catch (e: PackageManager.NameNotFoundException) {
            e.printStackTrace()
        }
        return version
    }
}